bucket/windows-application-driver.json

```
@@ -6,6 +6,10 @@
     "url": "https://github.com/Microsoft/WinAppDriver/releases/download/v1.2.1/WindowsApplicationDriver_1.2.1.msi",
     "hash": "a76a8f4e44b29bad331acf6b6c248fcc65324f502f28826ad2acd5f3c80857fe",
     "bin": "Windows Application Driver\\WinAppDriver.exe",
+    "post_install": "mv $dir\\System\\* $dir\\Windows*",
```

```suggestion
    "post_install": [
        "Move-Item \"$dir\\System\\*\", \"$dir\\Windows Application Driver\\*\" -Destination \"$dir\"",
        "Remove-Item \"$dir\\System\", \"$dir\\Windows Application Driver\""
    ],
```

Move all files to the app root directory.

---

This will conflict with the creation of a bin shim. I will remove the bin shim.